packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui' show DisplayFeature;
```

Comment:
Changes in this file look like they should be a separate change, they are different from the others that are adding the DisplayFeatureSubscreen and passing through the anchor point. Can you explain why this is different?

Reply:
I can separate this in a different PR. It is different because popup menus do some other calculations involving the edges of the screen. Some code is duplicated between this and DisplayFeatureSubScreen so I will try to refactor this to use DisplayFeatureSubScreen similarly to how dialogs do. If I can't, then I'll open a separate PR.

Reply:
Separated it into a separate PR https://github.com/flutter/flutter/pull/98981